doc/reference/common-weaknesses/off-chain-masking.rst

```diff
@@ -0,0 +1,15 @@
+On-chain checks may be masked by off-chain checks
```

Comment:
I think this is a huge issue and could do with more expansion. Could also link to the tweag cooked-validators work which is attempting to address this.